Application Lifecycle Management/instances-api.md

```diff
@@ -747,164 +1013,187 @@ ElasticBox-Release: 4.0
-   "readme": {
-     "url": "/resources/default_box_overview.md",
-     "upload_date": "2015-10-21 08:35:50.164926",
-     "length": 1302,
-     "content_type": "text/x-markdown"
-   },
-   "organization": "elasticbox",
-   "id": "e57466ee-7094-4bd4-9121-a6df4395d493",
-   "schema": "http://elasticbox.net/schemas/boxes/policy"
- },
- "created": "2015-10-29 12:26:20.377029",
- "uri": "/services/instances/i-3e43qa",
- "state": "done",
- "boxes": [
-   {
-     "updated": "2015-10-29 11:55:23.842461",
-     "automatic_updates": "off",
-     "requirements": [
+#### Structure
+```
+[PUT] /services/instances/{instance_id}
+```
 
-     ],
-     "description": "sample box",
-     "name": "ScriptBoxSample",
-     "created": "2015-10-29 10:52:08.446868",
-     "deleted": null,
-     "variables": [
-       {
-         "required": true,
-         "type": "Text",
-         "name": "variable_name",
-         "value": "variable_value",
-         "visibility": "public"
-       }
-     ],
-     "visibility": "workspace",
-     "id": "7a99f75c-30e6-4986-9059-f6889d1ff5f9",
-     "members": [
-
-     ],
-     "owner": "operations",
-     "organization": "elasticbox",
-     "events": {
-       "configure": {
-         "url": "/services/blobs/download/5631fa7614841250525226cc/configure",
-         "length": 5,
-         "destination_path": "scripts",
-         "content_type": "text/x-shellscript"
-       },
-       "install": {
-         "url": "/services/blobs/download/5631fa6614841250525226ca/install",
-         "length": 5,
-         "destination_path": "scripts",
-         "content_type": "text/x-shellscript"
-       }
-     },
-     "schema": "http://elasticbox.net/schemas/boxes/script"
-   }
- ],
- "schema": "http://elasticbox.net/schemas/instance",
- "members": [
-
- ],
- "owner": "operations",
- "variables": [
-   {
-     "required": true,
-     "type": "Text",
-     "name": "variable_name",
-     "value": "variable_value_overridden",
-     "visibility": "public"
-   }
- ],
- "operation": {
-   "event": "deploy",
-   "workspace": "operations",
-   "created": "2015-10-29 12:26:20.504356"
- },
- "id": "i-3e43qa",
- "icon": null
-}
+#### Example
+```
+[PUT] https://cam.ctl.io/services/instances/eb-bbhzh
 ```
 
-### PUT /services/instances/{instance_id}
-
-Given the instance ID, updates only these fields of an existing instance: boxes, tags, schedule, members, and variables. The request body should contain an instance object.
 
-**Normal Response Codes**
+### Request
 
-* 200
-* 202
+#### Headers
+```
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
 
-**Error Response Codes**
+#### URI Parameters
+|Parameter |Type |Description | Req. |
+|-----------|------|------------| ---- |
+| instance_id | string | Instance id | Yes |
 
-* Invalid Data (400)
-* Forbidden (403)
-* Not Found (404)
 
-**Request Parameters**
+#### Request Body Parameters
 
-| Parameter | Type |Description |
+|Parameter |Type |Description |
```

Comment:
Add in this table the Req. column too.
Then, in rows where necessary delete the 'Required' word and add yes in the new column.